dev-utils/poly-cli/index.js

```
@@ -146,18 +94,25 @@ function interactiveSetup() {
     inquirer
         .prompt(setup_questions)
         .then((answers) => {
-            checkIfValueExists("feature_name", answers);
+            checkIfValueExists("type", answers);
+            checkIfValueExists("name", answers);
             checkIfValueExists("author", answers);
             checkIfValueExists("version", answers);
             checkIfValueExists("description", answers);
             checkIfValueExists("license", answers);
-            setup(
-                answers.feature_name,
-                answers.author,
-                answers.version,
-                answers.description,
-                answers.license
-            );
+
+            if (answers.type === "empty") {
+                handleCreateEmptyFeature(answers);
+            } else if (answers.type === "preview") {
+                handleCreatePreviewFeature(answers);
+            } else if (answers.type === "importer") {
+                handleCreateImporterFeature();
+            } else {
+                printErrorMsg(
```

Nit: Same here, maybe just throw the error with descriptive message.